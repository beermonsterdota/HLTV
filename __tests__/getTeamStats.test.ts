import HLTV from '../src/index.js'
import { sleep } from '../src/utils.js'
import { expect, test } from 'vitest'

const FULL = 9863
const EMPTY = 9918

test('getTeamStats', async () => {
  await sleep(3000)
  expect(await HLTV.getTeamStats({ id: FULL })).toMatchSnapshot()
  await sleep(3000)
  expect(await HLTV.getTeamStats({ id: EMPTY })).toMatchSnapshot()
  await sleep(3000)
}, 500000)
