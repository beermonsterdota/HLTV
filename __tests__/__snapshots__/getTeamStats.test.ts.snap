// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`getTeamStats 1`] = `
{
  "currentLineup": [],
  "events": [],
  "historicPlayers": [],
  "id": 9863,
  "mapStats": {},
  "matches": [],
  "name": "",
  "overview": {
    "draws": undefined,
    "kdRatio": undefined,
    "losses": undefined,
    "mapsPlayed": undefined,
    "roundsPlayed": undefined,
    "totalDeaths": undefined,
    "totalKills": undefined,
    "wins": 0,
  },
  "standins": [],
  "substitutes": [],
}
`;

exports[`getTeamStats 2`] = `
{
  "currentLineup": [],
  "events": [],
  "historicPlayers": [],
  "id": 9918,
  "mapStats": {},
  "matches": [],
  "name": "",
  "overview": {
    "draws": undefined,
    "kdRatio": undefined,
    "losses": undefined,
    "mapsPlayed": undefined,
    "roundsPlayed": undefined,
    "totalDeaths": undefined,
    "totalKills": undefined,
    "wins": 0,
  },
  "standins": [],
  "substitutes": [],
}
`;
